### PR TITLE
Test: editor header component

### PR DESCRIPTION
**Pull request by rayzhou-bit**

### CODE REVIEW

File: src/editors/components/EditorHeader/HeaderTitle.test.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (rayzhou-bit) [inline src/editors/components/EditorHeader/HeaderTitle.test.jsx:1]: WIP

File: src/editors/components/EditorHeader/index.test.jsx [hunk 2]
```diff
@@ -1,10 +1,26 @@
 import React from 'react';
```
Comment (rayzhou-bit) [inline src/editors/components/EditorHeader/index.test.jsx:1]: WIP

**Comment by connorhaugh** [comment]
Note: make sure to remove any lint errors from files you edit as you work on them, so that we might move toward not having everything fail on lint errors.

File: src/editors/components/EditorHeader/HeaderTitle.test.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+}));
+
+describe('HeaderTitle', () => {
+  const props = {
+    editorRef: jest.fn().mockName('args.editorRef'),
+    isInitialized: false,
+    setBlockTitle: jest.fn().mockName('args.setBlockTitle'),
+    typeHeader: 'html',
+  };
+  const hookProps = {
```
Comment (connorhaugh) [inline src/editors/components/EditorHeader/HeaderTitle.test.jsx:33]: nit: perhaps consider renaming `hookprops`. That name implies that it is props for either the component, or the hook, when it is in fact the return value of the hook-- something we often refer to as a `state`. Just a nit though.

File: src/editors/containers/TextEditor/TextEditor.jsx [hunk 4]
```diff
@@ -40,7 +40,9 @@ export const TextEditor = ({
   blockFinished,
   initializeEditor,
 }) => {
-  console.log({ blockValue, blockFailed, blockFinished, test: 1 });
+  console.log({
```
Comment (connorhaugh) [inline src/editors/containers/TextEditor/TextEditor.jsx:43]: we can get rid of this console log.

**Review by connorhaugh — APPROVED** [review]
Nice-- this looks good. Just a couple of nits.

File: src/editors/components/EditorHeader/index.test.jsx [hunk 5]
```diff
@@ -1,58 +1,57 @@
...
-      <EditorHeader title={title} />
-    </EditorPageContext.Provider>,
-  );
-  expect(screen.getByText(title)).toBeTruthy();
-  expect(screen.getByLabelText('Close')).toBeTruthy();
-  userEvent.click(screen.getByLabelText('Close'));
-  expect(window.location.assign).toHaveBeenCalled();
+  let el;
+  beforeEach(() => {
+    el = shallow(<module.EditorHeader {...props} />);
```
Comment (muselesscreator) [inline src/editors/components/EditorHeader/index.test.jsx:35]: React is weird about trying to do compoent like this (lowercase).
Recommend `const { EditorHeader } = module;  el = shallow(<EditorHeader {...props} />);`
(note: the snapshot is undefined in the snap file)

File: src/setupTest.js [hunk 6]
```diff
@@ -58,7 +58,9 @@ jest.mock('@edx/frontend-platform/i18n', () => {
 });
 
 jest.mock('@edx/paragon', () => jest.requireActual('testUtils').mockNestedComponents({
-  ActionRow: 'ActionRow',
+  ActionRow: {
+    Spacer: 'ActionRow.Spacer',
```
Comment (muselesscreator) [inline src/setupTest.js:62]: ++

**Review by muselesscreator — CHANGES_REQUESTED** [review]
One snapshot update needed.  Thanks for adding the ActionRow Spacer to the testUtils :-)

**Review by muselesscreator — APPROVED** [review]
lgtm

**Pull request merged**